src/store/queue/sagas.ts

```diff
@@ -281,14 +283,53 @@ export function* watchRemove() {
   })
 }
 
+/**
+ * Set media session action handlers
+ * Reference: https://developer.mozilla.org/en-US/docs/Web/API/MediaSession/setActionHandler
+ * NOTE: in chrome there'a a hack in AudioStream where we do not call pause and thus play/pause
```

Comment:
this seems kinda unfortunate. i imagine you're talking about this https://github.com/AudiusProject/audius-client/blob/ace8ea441f8f8c3e4b4a934ead769dcef7869c1d/src/audio/AudioStream.ts#L334

is there any way around that or does play/pause not work entirely?

Reply:
If you check the html audio object, it never calls pause, so checking `audius.paused` always returns false.
So on when hitting the play/pause btn, it will always think it's playing even when it's paused, but when it's playing you can pause it.

Reply:
can we pull in the paused state from somewhere else? like a selector? i dont think we should ship if this is still borked